src/app/state/transaction/types.ts

```diff
@@ -9,7 +41,7 @@ export interface TransactionState {
   preview?: TransactionPreview
 }
 
-export type TransactionType = 'transfer' | 'addEscrow' | 'reclaimEscrow'
+export type NewTransactionType = 'transfer' | 'addEscrow' | 'reclaimEscrow'
```

Comment:
Not for this PR, but why we want to keep this NewTransactionType (reclaimEscrow vs reclaimrescrow in TransactionType)

Reply:
This is used for creating transactions, and didn't check how it is used yet. If it needs to be sent in this format we could replace this with another mapper :shrug: